code/SecurityAdmin.php

```diff
@@ -58,7 +58,7 @@ class SecurityAdmin extends ModelAdmin implements PermissionProvider
 
     private static $menu_priority = 0;
 
-    private static $tree_class = Group::class;
+    private static $model_class = Group::class;
```

Comment:
I strongly suspect this config doesn't actually do anything for this class, but checking that is out of scope for this PR. I'm just updating it to the config that `LeftAndMain` expects.